pkg/pillar/types/zedroutertypes.go

```diff
@@ -2311,6 +2318,8 @@ func (status NetworkInstanceStatus) LogModify(logBase *base.LogObject, old inter
 	}
 	// XXX remove?
 	logObject.CloneAndAddField("diff", cmp.Diff(oldStatus, status)).
+		AddField("vlan-port-info", fmt.Sprintf("%v", status.VlanMap)).
+		AddField("num-trunk-ports-int64", status.NumTrunkPorts).
```

Comment:
Since it dumps all diffs you don't need explicit fields. You might want explicit fields for some of the other log output like create and delete if that is useful.